### DISCUSSION TITLE
Slot component automatically added in component list

### DISCUSSION THREAD

**Discussion by azrlucky** [optimajet/formengine#41] [Q&A] [answered]
Hi,
The Slot component of type template is automatically added in our component list. How can we avoid this?
![image](https://github.com/user-attachments/assets/d285d06f-3731-445b-b177-efca046a33e0)

**Comment by optimajet** [comment, marked as answer]
The slot component is always available - it's part of the system. We have added the task of managing the list of visible components to our backlog.
